app/assets/javascripts/components/comment_box.js.jsx

```diff
@@ -0,0 +1,40 @@
+var CommentList = require('./comment_list');
+var CommentForm = require('./comment_form');
+var React = require('react');
+
+var data = [
+  {author: "Vicky Zhang", text: " This is one comment This is one comment " +
+  "This is one comment This is one comment This is one comment"},
+  {author: "Vicky Zhang", text: "This is one comment"},
+  {author: "Vicky Zhang", text: "This is one comment"},
+  {author: "Vicky Zhang", text: "This is one comment"},
+  {author: "Vicky Zhang", text: "This is one comment"},
+  {author: "Vicky Zhang", text: "This is one comment"},
+  {author: "Vicky Zhang", text: "This is one comment"},
+  {author: "Vicky Zhang", text: "This is one comment"},
+  {author: "Jordan Walke", text: "This is another comment"}
+];
+
+var CommentBox = React.createClass({
+
+  getInitialState: function() {
+    return {data: data};
```

Comment:
This is a perfect candidate for a CommentStore. See @15. We'll get the CommentStore changes in the next PR, though.